samples/v1.0/HostConfig/bing.json

```diff
@@ -1,121 +1,121 @@
-  "fontFamily": "Segoe UI",
-  "fontSizes": {
-    "small": 12,
-    "default": 14,
-    "medium": 17,
-    "large": 21,
-    "extraLarge": 26
-  },
-  "fontWeights": {
-    "lighter": 200,
-    "default": 400,
-    "bolder": 600
-  },
-  "containerStyles": {
-    "default": {
-      "backgroundColor": "#FFFFFFFF",
-      "foregroundColors": {
-        "default": {
-          "default": "#FF333333",
-          "subtle": "#EE333333"
-        },
-        "accent": {
-          "default": "#FF2E89FC",
-          "subtle": "#882E89FC"
-        },
-        "attention": {
-          "default": "#FFFF0000",
-          "subtle": "#DDFF0000"
-        },
-        "good": {
-          "default": "#FF54a254",
-          "subtle": "#DD54a254"
-        },
-        "warning": {
-          "default": "#FFc3ab23",
-          "subtle": "#DDc3ab23"
-        }
-      }
-    },
-    "emphasis": {
-      "backgroundColor": "#08000000",
-      "foregroundColors": {
-        "default": {
-          "default": "#FF333333",
-          "subtle": "#EE333333"
-        },
-        "accent": {
-          "default": "#FF2E89FC",
-          "subtle": "#882E89FC"
-        },
-        "attention": {
-          "default": "#FFFF0000",
-          "subtle": "#DDFF0000"
-        },
-        "good": {
-          "default": "#FF54a254",
-          "subtle": "#DD54a254"
-        },
-        "warning": {
-          "default": "#FFc3ab23",
-          "subtle": "#DDc3ab23"
-        }
-      }
-    }
-  },
-  "imageSizes": {
-    "small": 60,
-    "medium": 120,
-    "large": 180
-  },
-  "actions": {
-    "maxActions": 5,
-    "spacing": "default",
-    "buttonSpacing": 10,
-    "showCard": {
-      "actionMode": "inline",
-      "inlineTopMargin": 16
-    },
-    "actionsOrientation": "vertical",
-    "actionAlignment": "stretch"
-  },
-  "adaptiveCard": {
-    "allowCustomStyle": false
-  },
-  "imageSet": {
-    "imageSize": "medium",
-    "maxImageHeight": 100
-  },
-  "factSet": {
-    "title": {
-      "color": "default",
-      "size": "default",
-      "isSubtle": false,
-      "weight": "bolder",
-      "wrap": true,
-      "maxWidth": 150
-    },
-    "value": {
-      "color": "default",
-      "size": "default",
-      "isSubtle": false,
-      "weight": "default",
-      "wrap": true
-    },
-    "spacing": 10
-  }
+	"spacing": {
+		"small": 3,
+		"default": 8,
+		"medium": 20,
+		"large": 30,
+		"extraLarge": 40,
+		"padding": 10
+	},
+	"separator": {
+		"lineThickness": 1,
+		"lineColor": "#FFEEEEEE"
+	},
+	"supportsInteractivity": true,
+	"fontFamily": "Segoe UI",
+	"fontSizes": {
+		"small": 12,
+		"default": 14,
+		"medium": 17,
+		"large": 21,
+		"extraLarge": 26
+	},
+	"fontWeights": {
+		"lighter": 200,
+		"default": 400,
+		"bolder": 600
+	},
+	"containerStyles": {
+		"default": {
+			"backgroundColor": "#FFFFFFFF",
+			"foregroundColors": {
+				"default": {
+					"default": "#FF333333",
+					"subtle": "#EE333333"
+				},
+				"accent": {
```

Comment:
Some HostConfigs like this one are missing the light and dark color definitions.

Reply:
I'll just delete it, it's old and not used right now